rocAL/rocAL/include/readers/image/caffe2_lmdb_record_reader.h

```diff
@@ -67,6 +67,11 @@ class Caffe2LMDBRecordReader : public Reader
     int close() override;
 
     Caffe2LMDBRecordReader();
+    //! return feed_data: not implemented
+    void feed_file_names(const std::vector<std::string>& file_names, size_t num_images, bool eos=false) override {return;}
+
+    //! return feed_data: not implemented
+    void feed_data(const std::vector<unsigned char *>& images, const std::vector<size_t>& image_size, int mode, bool eos = false, int width=0, int height=0, int channels=0) override{return;}
```

Comment:
Please add space before and after return; 
Applicable to all such declarations for this function

Reply:
Done